public/docs/dart/latest/tutorial/toh-pt6.jade

```diff
@@ -1 +1,139 @@
-!= partial("../../../_includes/_ts-temp")
+extends ../../../ts/latest/tutorial/toh-pt6.jade
+
+block includes
+  include ../_util-fns
+  - var _Http = 'BrowserClient';
+  - var _Angular_Http = 'Dart <code>BrowserClient</code>'
+  - var _httpUrl = 'https://pub.dartlang.org/packages/http'
+  - var _Angular_http_library = 'Dart <a href="' + _httpUrl + '"><b>http</b></a> library'
+  - var _HTTP_PROVIDERS = 'BrowserClient'
+  - var _JSON_stringify = 'JSON.encode'
+
+block start-server-and-watch
+  :marked
+    ### Keep the app compiling and running
+    Open a terminal/console window.
+    Start the Dart compiler, watch for changes, and start our server by entering the command:
+
+  code-example(language="bash").
+      pub serve
+ 
+block http-library
+  :marked
+    We'll be using the !{_Angular_Http} to communicate with a server.
+    This class is exported by the !{_Angular_http_library}.
+
+    ### Pubspec updates
+    
+    We need to add a package dependency for the !{_Angular_http_library}.
+
+    We also need to inform the Dart Angular 2 Transformer that we will be using
+    `BrowserClient` (we'll defer explaining _why_ this extra configuration is
+    needed until the [HTTP client chapter](../guide/server-communication.html#!#http-providers))
+    by means of the transformer's `resolved_identifiers` configuration parameter.
+    Since we will also be needing the !{_Angular_http_library} `Client` interface, we'll
+    add that now as well.
+
+    Update `pubspec.yaml` to look like this (additions are highlighted):
+
+  - var stylePattern = { pnk: /(http.*|resolved_identifiers:|Browser.*|Client.*)/gm };
+  +makeExcerpt('pubspec.yaml', 'additions', null, stylePattern)
+
+block http-providers
+  :marked
+    Before our app can use `#{_Http}`, we have to register it as a service provider.
+
+block backend
+  :marked
+    We want to replace `BrowserClient`, the service that talks to the remote server,
+    with the in-memory web API service. 
+    Our in-memory web API service, shown below, is implemented using the
+    standard `http` library `MockClient` class.
+    All `http` client implementations share a common `Client` interface, so
+    we'll have our app use the `Client` type so that we can freely switch between
+    implementations.
+
+block dont-be-distracted-by-backend-subst
+  //- N/A
+
+block get-heroes-details
+  :marked
+    We make an asynchronous call to `http.get()` which returns an HTTP
+    `Response`.  The response payload (`body`) is decoded as JSON via the
+    `_extractData` helper method.
+
+block hero-detail-comp-extra-imports-and-vars
+  //- N/A
+
+block hero-detail-comp-updates
+  :marked
+    ### Edit in the *HeroDetailComponent*
```

Comment:
"Edit in" sounds weird. (the TS "Add/Edit in", strangely, seems more understandable, probably because you have more context.)

Why are the Dart & TS versions so different here?